install_config/configuring_sdn.adoc

```diff
@@ -154,12 +154,21 @@ for any changes to take effect.
 
 [IMPORTANT]
 ====
-The `*serviceNetworkCIDR*` and `*hostSubnetLength*` values cannot be changed
-after the cluster is first created, and `*clusterNetworkCIDR*` can only be
-changed to be a larger network that still contains the original network. For
-example, given the default value of *10.128.0.0/14*, you could change
-`*clusterNetworkCIDR*` to *10.128.0.0/9* (i.e., the entire upper half of net
-10) but not to *10.64.0.0/16*, because that does not overlap the original value.
+The `*hostSubnetLength*` value cannot be changed after the cluster is
+first created, `*clusterNetworkCIDR*` can only be changed to be a
+larger network that still contains the original network, and
+`*serviceNetworkCIDR*` can only be expanded. For example, given the
+default value of *10.128.0.0/14*, you could change
+`*clusterNetworkCIDR*` to *10.128.0.0/9* (i.e., the entire upper half
+of net 10) but not to *10.64.0.0/16*, because that does not overlap
+the original value.
+
+You could change `*serviceNetworkCIDR*` from *172.30.0.1/16* to
+*172.30.0.1/15*, but not to *172.28.0.1/14* because even though the
```

Comment:
Should be `172.30.0.0/16` `172.30.0.0/15` and `172/28.0.0/14`?

Reply:
Yes, thank you!  Fixed.